ios/RNTwilioVoice/RNTwilioVoice.m

```diff
@@ -681,6 +681,14 @@ - (void)provider:(CXProvider *)provider performSetMutedCallAction:(CXSetMutedCal
     }
 }
 
+- (void)provider:(CXProvider *)provider performPlayDTMFCallAction:(CXPlayDTMFCallAction *)action {
+  TVOCall *call = self.activeCalls[action.callUUID.UUIDString];
+  if (call && call.state == TVOCallStateConnected) {
+    RCTLogInfo(@"SendDigits %@", action.digits);
```

Comment:
We haven't used `RCTLogInfo()` so far. I would suggest to stick to `NSLog()`.
If you see advantages that I am not aware of by using `RCTLogInfo()`, please open another PR that changes all the occurrences of `NSLog()`